cmd/auth/login.go

```diff
@@ -140,6 +141,9 @@ depends on the existing profiles you have set in your configuration file
 			AuthType:  "databricks-cli",
 			ClusterID: clusterID,
 		}
+		if os.Getenv("DATABRICKS_CONFIG_FILE") != "" {
```

Comment:
would it make sense to get rid of the second os call by caching the value to a local variable?

Reply:
Yes, make sense, fixed